### PR TITLE
[globalcache] Add support for Zmote ZV-2 universal remote.

### PR DESCRIPTION
**Pull request by mhilbush**
Signed-off-by: Mark Hilbush <mark@hilbush.com>

### CODE REVIEW

File: addons/binding/org.openhab.binding.globalcache/ESH-INF/thing/channels.xml [hunk 1]
```diff
@@ -86,6 +86,21 @@
 		</channels>
 	</channel-group-type>
 
+    <channel-group-type id="ir-m1-c1">
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.globalcache/ESH-INF/thing/channels.xml]: Could you run the auto-formatter, please?

File: addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/GlobalCacheBindingConstants.java [hunk 2]
```diff
@@ -37,11 +37,12 @@
     public final static ThingTypeUID THING_TYPE_ITACH_FLEX = new ThingTypeUID(BINDING_ID, "itachFlex");
     public final static ThingTypeUID THING_TYPE_GC_100_06 = new ThingTypeUID(BINDING_ID, "gc100_06");
     public final static ThingTypeUID THING_TYPE_GC_100_12 = new ThingTypeUID(BINDING_ID, "gc100_12");
+    public final static ThingTypeUID THING_TYPE_ZMOTE = new ThingTypeUID(BINDING_ID, "zmote");
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/GlobalCacheBindingConstants.java]: Can you rebase on the current master, fix the merge conflicts and make the order of modifiers for these new lines also `public static final`

Comment (mhilbush) [inline addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/GlobalCacheBindingConstants.java]: Sure.  I'm a bit of a git newb, so I may have a question or two along the way.  :-)

Comment (martinvw) [inline addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/GlobalCacheBindingConstants.java]: Np

Comment (mhilbush) [inline addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/GlobalCacheBindingConstants.java]: Well, that didn't go so well, and it'll probably take even more time to sort it out.  Any issue with just closing this PR and submitting another?

**Comment by martinvw** [comment]
@mhilbush I just merged it online using some github function, it seems okay, can you check?

**Comment by mhilbush** [comment]
This looks good, @martinvw.  Thank you for doing that!

The way I did it...  I ended up with a new local branch that contained current master, my changes from this PR, the resolution of merge conflicts, and the correction of `final static` (which is what we wanted).  What I wasn't sure about was pushing that new branch so that it replaced what was in this PR.

**Comment by mhilbush** [comment]
Thanks @martinvw!!!

File: addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/discovery/GlobalCacheDiscoveryService.java [hunk 3]
```diff
@@ -165,13 +165,12 @@ private synchronized void discover() {
...
                             && discoveryServiceCallback.getExistingDiscoveryResult(uid) == null
                             && discoveryServiceCallback.getExistingThing(uid) == null) {
-
                         logger.trace("Creating discovery result for: {}, type={}, IP={}", uid,
                                 gcMulticastListener.getModel(), gcMulticastListener.getIPAddress());
 
                         thingDiscovered(DiscoveryResultBuilder.create(uid)
                                 .withProperties(properties)
-                                .withLabel("GlobalCache " + gcMulticastListener.getModel())
+                                .withLabel(gcMulticastListener.getVendor() + " " + + gcMulticastListener.getModel())
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/discovery/GlobalCacheDiscoveryService.java]: This line broke the build - https://github.com/openhab/openhab2-addons/pull/2172 fixes it again.
Will have to figure out why the pull request builder of Travis+Cloudbees isn't working anymore ATM :-(

**Pull request merged**